c/include/libsbp/ext_events.h

```diff
@@ -35,6 +35,33 @@ SBP_PACK_START
  * which pin it was and whether it was rising or falling.
  */
 #define SBP_MSG_EXT_EVENT 0x0101
+#define SBP_EXT_EVENT_TIME_QUALITY_MASK (0x1)
+#define SBP_EXT_EVENT_TIME_QUALITY_SHIFT (1u)
+#define SBP_EXT_EVENT_TIME_QUALITY_GET(flags) \
+                             (((flags) >> SBP_EXT_EVENT_TIME_QUALITY_SHIFT) \
+                             & SBP_EXT_EVENT_TIME_QUALITY_MASK)
+#define SBP_EXT_EVENT_TIME_QUALITY_SET(flags, val) \
+                             do {(flags |= \
+                             ((val & SBP_EXT_EVENT_TIME_QUALITY_MASK) \
+                             << SBP_EXT_EVENT_TIME_QUALITY_SHIFT));} while(0);
```

Comment:
Per CERT C recommendations: https://wiki.sei.cmu.edu/confluence/display/c/PRE11-C.+Do+not+conclude+macro+definitions+with+a+semicolon -- a macro should not conclude with a semicolon

Reply:
I think that was an accident; thanks for noticing!